onadata/libs/serializers/project_serializer.py

```diff
@@ -369,6 +369,15 @@ def validate(self, attrs):
             })
         return attrs
 
+    def validate_public(self, value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5585241)